### DISCUSSION TITLE
Running Quspin on multiple nodes of HPC

### DISCUSSION THREAD

**Discussion by magnon00** [QuSpin/QuSpin#459] [Q&A]
Is it possible to run Quspin on multiple nodes of a High Performance cluster (HPC) ?

I was told by in-charge of HPC that this package cannot be run on multiple nodes of HPC
 though can run on single node of HPC.

**Comment by mgbukov** [comment]
quspin has some built-in omp parallelization, which you can make use of. OMP spawns multiple threads within the same process. See [this tutorial](https://weinbe58.github.io/QuSpin/parallelization.html) for details. 

If you want to distribute your work among multiple processes, quspin's functions can't do this automatically. But you might be abe to use [mpi4py](https://mpi4py.readthedocs.io/en/stable/) and distribute the task yourself.

**Reply by magnon00** [reply]
It will be great and very helpful for beginners if one example can be shown demonstrating the distribution of quspin script among multiple processes.
